### PR TITLE
VZ-3517: Automate scanning Verrazzano release artifacts and creating the github release

### PR DESCRIPTION
**Pull request by bhatpmk**
# Description

Automates the steps of downloading the Verrazzano release artifacts, scanning them and creating a Github release.

Fixes VZ-3517

# Checklist 

As the author of this PR, I have:

- [x] Checked that I included or updated copyright and license notices in all files that I altered
- [ ] Added or updated unit tests for any new functions I added
- [ ] Added or updated integration tests if appropriate
- [ ] Added or updated acceptance tests if appropriate

Code reviewer, please confirm this PR:

- [ ] Addressed the requirement and meets the acceptance criteria
- [ ] Does not introduce unrelated or spurious changes
- [ ] Does not introduce any unapproved dependency
- [ ] Makes sense and it easy to understand, and/or difficult areas of code are clearly documented so that they can be understood

### CODE REVIEW

File: release/scripts/common.sh [hunk 1]
```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
```
Comment (fredtibbitts) [inline release/scripts/common.sh:1]: Nit, we've been using `#!/usr/bin/env bash` for release scripts and I see two of your other scripts use it. I think we should be consistent and use `#!/usr/bin/env bash`.

File: release/scripts/get_release_artifacts.sh [hunk 2]
```diff
@@ -0,0 +1,63 @@
+#!/bin/bash
```
Comment (fredtibbitts) [inline release/scripts/get_release_artifacts.sh:1]: `#!/usr/bin/env bash`

**Review by fredtibbitts — COMMENTED** [review]
Just the two minor comments, otherwise this looks good to me.

File: release/scripts/get_release_artifacts.sh (hunk 2 above)
Comment (bhatpmk) [reply release/scripts/get_release_artifacts.sh]: I have changed the script.

File: release/scripts/common.sh (hunk 1 above)
Comment (bhatpmk) [reply release/scripts/common.sh]: I have changed the script.

**Review by fredtibbitts — APPROVED** [review]

**Review by agv426 — APPROVED** [review]

**Pull request merged**